File: user-service/src/main/java/rs/edu/raf/model/PaymentCode.java
package rs.edu.raf.model;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
public class PaymentCode {
    @Id
    @GeneratedValue (strategy = GenerationType.IDENTITY)
    private Long id;
    private Integer formAndBasis;
    private String paymentDescription;

}

File: user-service-kotlin/src/main/kotlin/org/example/userservicekotlin/model/PaymentCode.kt
package org.example.userservicekotlin.model

import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.Id

@Entity
class PaymentCode(
    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
    val id: Long,
    val formAndBasis: String,
    val paymentDescription: String
) {
}